src/notation/internal/notationinteraction.cpp

```diff
@@ -734,9 +734,7 @@ void NotationInteraction::doSelect(const std::vector<EngravingItem*>& elements,
         }
     }
 
-    for (EngravingItem* element: elements) {
-        score()->select(element, type, staffIndex);
-    }
+    score()->select({ elements }, type, staffIndex);
```

Comment:
Why is `{ elements }` here between braces? Is it to deliberately copy the vector before passing it to `score()->select(…)`, or is it a typo? :)

Reply:
thank you! fixed